### ISSUE TITLE
[Bug]: Mistake with method definition

### ISSUE THREAD

**Issue by moez-c180** [savannabits/filament-flatpickr#69]
### What happened?

There is an error when I use the Flatpickr component because the definition of minDate is not correct:

```
public function minDate(Carbon|string|null $minDate|Closure): static
{
        $this->minDate = $minDate ? Carbon::parse($minDate) : $minDate;

         return $this;
}

```
The error is with the line of: public function minDate(Carbon|string|null $minDate|Closure): static

It should be: public function minDate(Carbon|string|null|Closure $minDate): static

### How to reproduce the bug

Just initiate the component in any filament form

### Package Version

3

### PHP Version

8.2

### Laravel Version

11

**Comment by holmesadam** [comment]
I'm also having this issue with a fresh install - manually edited the vendor file whilst waiting for this to be fixed. 

Any idea when you can fix this @coolsam726? 

Thanks

**Comment by munchy5132** [comment]
Same here. Any update on this ?

**Comment by holmesadam** [comment]
> Same here. Any update on this ?

I ended up switching to dev branch

`"savannabits/filament-flatpickr": "^3.x-dev,`

**Comment by vaskaraminashvili** [comment]
same here

**Comment by alboxe** [comment]
same here. career-180 shows the solution: public function minDate(Carbon|string|null|Closure $minDate): static
Why is this little adjustement not applied to the latest version?

**Comment by CodeDanger** [comment]
try to replace method definiation from
    public function minDate(Carbon|string|null $minDate|Closure): static
to
    public function minDate(Carbon|string|null|Closure $minDate): static
that's will work for you

**Comment by BrendanBasson4** [comment]
same here

**Comment by coolsam726** [comment]
Sorry guys, I have been quite swamped up. The fix is now released in 3.1.1
Thanks for the PRs.
